scripts/script.js

```diff
@@ -69,6 +76,9 @@ cocktailApp.getRandomCocktail = function() {
     .catch((err) => {
       console.log(`Our drink monkeys couldn't find anything. Sorry. :(`, err);
     });
+    
+    cocktailApp.popup.classList.remove('popup__container--open');
```

Comment:
The popup__container has both an _open_ and a _closed_ class while it's visible.

So that the popup isn't both open and closed at the same time (and because we already have --hidden modifiers elsewhere), I would suggest starting the popup with classes like popup__container (has all of the container styling) and popup__container--hidden (contains visibility: hidden or display: none). Then, you could remove the --hidden class to display the popup and add it back to hide it again. What do you think?